standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -555,6 +555,9 @@ public static ConfVars getMetaConf(String name) {
     EVENT_DB_LISTENER_TTL("metastore.event.db.listener.timetolive",
         "hive.metastore.event.db.listener.timetolive", 86400, TimeUnit.SECONDS,
         "time after which events will be removed from the database listener queue"),
+    CLEAN_MAX_EVENTS("metastore.event.db.clean.maxevents",
```

Comment:
EVENT_ prefix is used for other related configs. Can use the same here as well.

Reply:
Done.